### DISCUSSION TITLE
对于当下一些问题的总结

### DISCUSSION THREAD

**Discussion by XiFeng-exe** [Melledy/Grasscutter#1168] [General]
这里是对于当下一些问题的总结。感谢各位大佬提供的解决方案。
1.不能普通攻击和技能，你可以更新一下resource试试
2.logo后直接白屏，先确定你是不是之前进去下载过资源，是，那就启动器-设置-下载资源清除。否，那就不知道了，待补充。
3.bat报错，可以尝试更新项目文件和jar文件，同时删除config.json文件。然后重新按流程启动试试。
4.开门后白屏，有一定的可能是因为 3. 可以按照 3. 的步骤试试。
5.输入称呼后断开，你可以更新一下resource试试
6.进去后要下载资源，然后在21.8%左右会弹出code-9176？（好像是这个错误代码）的，更新resource，更新项目文件，最好还更新一下jar文件，同时删除config.json文件。
目前能想到的就这么多了，欢迎各位大佬在下面补充。
然后，我不是什么技术能人，我只是总结了一下近来看到的问题，各位大佬轻点喷。

**Comment by yoimiya43** [comment]
大兄弟，问一下我这个dev的构架有没有毛病？

1.下载grasscutter-1.1.2-dev.jar和和source code.zip，解压后得到 Grasscutter-1.1.2-dev 这个文件夹，把grasscutter-1.1.2-dev.jar丢进去。
2.在文件夹根目录运行这两个指令
.\gradlew.bat
.\gradlew jar
3.解压最新的resources文件到根目录，创建start.bat运行grasscutter-1.1.2-dev.jar

然后就是正常启动，一切正常，也没下载资源，就是开门后白屏。

**Reply by XiFeng-exe** [reply]
这可能是grasscutter-1.1.2-dev.jar文件的问题，你去这个地方在吧，https://www.grasscutter.ga/index.html#download，下载Development Version 最新的，然后删除config.json，然后再试试看

**Reply by yoimiya43** [reply]
换了最新的那个jar可以进游戏了谢谢
现在技能和攻击不行 我去换resource看看

**Reply by yoimiya43** [reply]
> 这可能是grasscutter-1.1.2-dev.jar文件的问题，你去这个地方在吧，[https://www.grasscutter.ga/index.html#download，下载Development](https://www.grasscutter.ga/index.html#download%EF%BC%8C%E4%B8%8B%E8%BD%BDDevelopment) Version 最新的，然后删除config.json，然后再试试看

我重新去这里下了resource文件夹 https://github.com/Koko-boya/Grasscutter_Resources，然后现在登录不上了。。显示连接失败，稍后再试。cmd提示的时候 requiry_region_list

**Comment by basaka2022** [comment]
jar都是编译好的，干嘛还解开

**Comment by wzq279** [comment]
启动后提示
<WARN:HttpServer> [Dispatch] 无法连接路由
